ibu-imager/internal/seedcreator/seedcreator.go

```diff
@@ -155,7 +156,9 @@ func (s *SeedCreator) stopServices() error {
 
 	s.log.Println("Stopping containers and CRI-O runtime.")
 	crioSystemdStatus, err := s.ops.SystemctlAction("is-active", "crio")
-	if err != nil {
+	var exitErr *exec.ExitError
+	// If ExitCode is 3, the command succeeded and told us that crio is down
+	if err != nil && errors.As(err, &exitErr) && exitErr.ExitCode() != 3 {
```

Comment:
This PR needs to be rebased for https://github.com/openshift-kni/lifecycle-agent/pull/38